connector_prestashop_catalog_manager/models/product_brand/common.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+from odoo.addons.component_event import skip_if
+
+
+class PrestashopProductBrandListener(Component):
```

Comment:
It does not work brand export because table prestashop_product_brand is not created.

Maybe this need some more classes.

Reply:
Good day @cubells, the class is available at branch https://github.com/hailangvn/connector-prestashop/tree/14.0-imp-connector_prestashop-product_brand. I was requested to remove product brand from connector_prestashop.